tests/performance/log-results.js

```diff
@@ -20,42 +21,66 @@ const resultsFiles = [
 	},
 ];
 
-const performanceResults = resultsFiles.map( ( { file } ) =>
-	JSON.parse(
-		fs.readFileSync( path.join( __dirname, '/specs/' + file ), 'utf8' )
+// Results files recorded for the base commit.
+const baseResultsFiles = [
+	{
+		file: 'base-home-block-theme.test.results.json',
+		metricsPrefix: 'home-block-theme-',
+	},
+	{
+		file: 'base-home-classic-theme.test.results.json',
+		metricsPrefix: 'home-classic-theme-',
+	},
+];
```

Comment:
Rather than having those two hard-coded lists here, I think a more maintainable solution would be to iterate through the files in `tests/performance/specs` and compute this data dynamically. We can compute both file names and the `metricsPrefix` based on the file name of the test spec file.

At a minimum, I would suggest to combine the two lists here into one, no need to duplicate the `metricsPrefix` entry. We could use one array which contains objects with `file`, `baseFile`, and `metricsPrefix`.

Reply:
Thanks @felixarntz for the review. I have implemented the changes https://github.com/10up/wordpress-develop/pull/36/commits/167250397ea8d8be8508fb21773244aad48af36e can you please review it so i can merge the PR?

cc. @joemcgill